docs/similarsoftware.rst

```diff
@@ -0,0 +1,97 @@
+#################
+Similar Software
+#################
+
+.. contents:: Contents
+
+Alternate MIP Solvers (with Python interfaces)
+==============================================
+
+In the following we will give a list of other mixed-integer optimizers with available python interfaces.
+As each solver has its own set of problem classes that it can solve we will use a table with reference
+keys to summarise these problem classes.
+
+.. note:: This table is by no means complete.
+
+.. note:: SCIP can solve of all the below problem classes (and many more).
+
+.. list-table:: Label Summaries
+  :widths: 25 25
+  :align: center
+  :header-rows: 1
+
+  * - Key
+    - Feature
+  * - LP
+    - Linear Programs
+  * - MILP
+    - Mixed-Integer Linear Programs
+  * - QP
+    - Quadratic Programs
+  * - MIQP
+    - Mixed-Integer Quadratic Programs
+  * - QCP
+    - Quadratically Constrained Programs
+  * - MIQCP
+    - Mixed-Integer Quadratically Constrained Programs
+  * - MINLP
+    - Mixed-Integer Nonlinear Programs
+  * - PB
+    - Pseudo-Boolean Problems
+  * - SOCP
+    - Second Order Cone Programming
+  * - SDP
+    - Semidefinite Programming
+  * - MISDP
+    - Mixed-Integer Semidefinite Programming
+
+Open Source
+***********
+
+- `HiGHS <https://github.com/ERGO-Code/HiGHS>`_: O, LP, MILP, QP, MIQP
+- `CLP / CBC <https://github.com/coin-or/CyLP>`_: O, LP, MILP
+- `GLOP <https://github.com/google/or-tools>`_: O, LP
+
+Closed Source
+*************
+
+- `CPLEX <https://www.ibm.com/products/ilog-cplex-optimization-studio>`_: LP, MILP, QP, MIQP, QCP, MIQCP, SOCP
+- `Gurobi <https://www.gurobi.com/>`_: LP, MILP, QP, MIQP, QCP, MIQCP, MINLP, PB, SOCP
+- `Xpress <https://www.fico.com/en/products/fico-xpress-optimization>`_: LP, MILP, QP, MIQP, QCP, MIQCP, MINLP, SOCP
+- `COPT <https://www.copt.de/>`_: LP, MILP, QP, MIQP, QCP, MIQCP, SOCP, SDP
+- `MOSEK <https://www.mosek.com/>`_: LP, MILP, QP, MIQP, QCP, MIQCP, SOCP, SDP, MISDP
+
+General Modelling Frameworks (Solver Agnostic)
+==============================================
+
+This list will contain general modelling frameworks that allow you to use swap out SCIP for other
```

Comment:
"allow you to use swap out SCIP" - I don't know if this is gramatically correct.

Also, maybe add AMPL here as well?

Reply:
I wanted to keep this Python specific. So I skipped a lot of solvers and frameworks

Reply:
TIL Ampl has a Python interface. I will add it